src/sentry/integrations/jira/client.py

```diff
@@ -215,3 +217,8 @@ def transition_issue(self, issue_key, transition_id):
     def assign_issue(self, key, name_or_account_id):
         user_id_field = self.user_id_field()
         return self.put(self.ASSIGN_URL % key, data={user_id_field: name_or_account_id})
+
+    def get_email(self, name_or_account_id):
+        user_id_field = self.user_id_field()
+        user = self.get_cached(self.EMAIL_URL, params={user_id_field: name_or_account_id})
```

Comment:
Should you have a try/except here to handle errors when Jira isn't available? Right now it looks like we would drop a webhook/fail a sync.

Reply:
@markstory If this API call fails, we return a 500 on the webhook. Jira doesn't retry the webhook as far as I can tell. I think that's a good way to handle the failure, yes? Not sure what else we can do.

Reply:
Jira retrying the webhook is a good process :+1:

Reply:
I took another look at our Celery task and it will also retry up to 5 times on should fetching an email fail.